tide_core.install

```diff
@@ -1082,15 +1082,3 @@ function tide_core_update_8045() {
     user_role_grant_permissions(Role::load($role)->id(), $permissions);
   }
 }
-
-/**
- * Enable tide_edit_protection module.
- */
-function tide_core_update_8046() {
```

Comment:
@vincent-gao this update hook was released in 2.0.8 so you can't remove it. The body of the hook can be removed. but the hook itself must stay.

Reply:
@sonnykt  sure. indeed it should keep same number with database.